packages/client/CHANGELOG.md

```diff
@@ -10,6 +10,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- Proxy enhancements: It is possible to set a limit for the number of proxy connections while having a larger set of nodes to choose from
```

Comment:
Could mention that there is an change in the API: 
- use `.setProxies` instead of `.openProxyConnections` and `.closeProxyConnections`

How difficult it would be to support the old methods temporarily so that we don't need to do a breaking change? We'd deprecate the methods, but they would work ok (just delegating the call to `setProxies`?)